pkgs/os-specific/linux/esdm/default.nix

```
@@ -59,16 +59,17 @@ assert cryptoBackend == "openssl" || cryptoBackend == "botan" || cryptoBackend =
 
 stdenv.mkDerivation rec {
   pname = "esdm";
-  version = "1.1.1";
+  version = "1.2.0";
 
   src = fetchFromGitHub {
     owner = "smuellerDD";
     repo = "esdm";
     rev = "v${version}";
-    hash = "sha256-Z8cIjNI+Qi6O2e72vbEefbCCXyIA+lcEMDzWJReGrUs=";
+    hash = "sha256-5XctrI02pfCgK1P76AaSkMjiQqav6LX3SMjKr4F44sw=";
   };
 
   nativeBuildInputs = [ meson pkg-config ninja ];
+
   buildInputs = [ protobufc ]
     ++ lib.optional (cryptoBackend == "botan" || botanRng) botan3
```

```suggestion
  buildInputs =
    lib.optional (cryptoBackend == "botan" || botanRng) botan3
```

I think it can be dropped from regular `buildInputs` when its available from `propagatedBuildInputs` now

---

Indeed. Thanks for the hint!